mlir/utils/jenkins/Jenkinsfile

```diff
@@ -1518,13 +1542,13 @@ pipeline {
                                                                     sh 'rm -f build/CMakeCache.txt'
                                                                     sh 'rm -f build/*.profraw'
                                                                     buildProject('check-rocmlir-build-only',
-                                                                                '-DBUILD_FAT_LIBROCKCOMPILER=ON -DCMAKE_BUILD_TYPE=debug -DLLVM_BUILD_INSTRUMENTED_COVERAGE=ON')
+                                                                                 '-DBUILD_FAT_LIBROCKCOMPILER=ON -DCMAKE_BUILD_TYPE=debug -DLLVM_BUILD_INSTRUMENTED_COVERAGE=ON')
                                                                     dir ('build') {
                                                                         // Run tests.
                                                                         collectCoverageData("${LLVM_PROFDATA}", "${LLVM_COV}", "${CODEPATH}")
                                                                         // Upload to codecov.
                                                                         withCredentials([string(credentialsId: 'codecov-token-rocmlir',
-                                                                                                variable: 'CODECOV_TOKEN')]) {
+                                                                                               variable: 'CODECOV_TOKEN')]) {
```

Comment:
Lot of whitespace changes. Please minimize those in future. Or else if you have any formatter for Jenkins, then make sure it is being applied automatically for the PRs.